README.md

```diff
@@ -69,17 +69,7 @@ Install all the wheel files from SDK's lib directory.
 ```cmd
 pip install -U lib/*/*.whl
 ```
-Install dependencies like *lxml* and *pyvmomi* from PyPI as other requirements were installed from SDK's lib directory.
-```cmd
-pip install -U <SDK_DIRECTORY_PATH>
-```
-Where <SDK_DIRECTORY_PATH> is either install directory of the SDK or location of SDK's zip
-e.g.
-```
-pip install -U /root/vsphere-automation-sdk-python
-Or
-pip install -U vsphere-automation-sdk-python-8.0.0.0.zip
-```
```

Comment:
What is the motivation for removing these steps? It is not clear from the commit message(s).

Reply:
Not removed the steps,  updated the location of the  SDK's zip path